README.md

```diff
@@ -4,9 +4,9 @@
 )](https://ci.appveyor.com/project/procxx/tdesktop)
 
 
-# [Telegram Desktop][telegram_desktop] - pro.cxx fork
+# [Kepka][telegram_desktop] - Unofficial [Telegram Desktop][telegram_desktop] fork from pro.cxx community
```

Comment:
I suggest to remove the first link from this line. "Kepka" shouldn't be a link to https://desktop.telegram.org/